### PR TITLE
Update account-permissions-and-security-settings-in-sharepoint-2013.md

### PR DESCRIPTION
**Pull request by tseward**
Fix for #810

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/install/account-permissions-and-security-settings-in-sharepoint-2013.md [hunk 1]
```diff
@@ -44,19 +43,17 @@ One of the following SharePoint components automatically configures most of the
...
+### Farm administrator user  account
 
-This account is used to set up each server in your farm by running the SharePoint Configuration Wizard, the initial Farm Creation Wizard, and PowerShell. For the examples in this article, the setup user administrator account is used for farm administration, and you can use Central Administration to manage it. Some configuration options, for example, configuration of the SharePoint 2013 Search query server, require local administration permissions. The setup user administrator account requires the following permissions:
+This account is a uniquely identifiable account assigned to the SharePoint administrator and is used to set up each server in your farm by running the SharePoint Configuration Wizard, the initial Farm Creation Wizard, and PowerShell. The account must be a domain user. For the examples in this article, the farm administrator account is used for farm administration, and you can use Central Administration to manage it. Some configuration options such as configuration of the SharePoint 2013 Search query server, require local administration permissions. The farm administrator user account requires the following permissions:
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/account-permissions-and-security-settings-in-sharepoint-2013.md]:
```suggestion
This account is a uniquely identifiable account assigned to the SharePoint administrator and is used to set up each server in your farm by running the SharePoint Configuration Wizard, the initial Farm Creation Wizard, and PowerShell. The account must be a domain user. For the examples in this article, the farm administrator account is used for farm administration, and you can use Central Administration to manage it. Some configuration options such as configuration of the SharePoint 2013 Search query server require local administration permissions. The farm administrator user account requires the following permissions:
```

File: SharePoint/SharePointServer/install/account-permissions-and-security-settings-in-sharepoint-2013.md [hunk 2]
```diff
@@ -44,19 +43,17 @@ One of the following SharePoint components automatically configures most of the
...
-- It must be a member of the local administrators group on each server in the SharePoint farm.
+- It must be a member of the Local Administrators group on each server in the SharePoint farm.
     
 - This account must have access to the SharePoint databases.
     
-- If you use any PowerShell operations that affect a database, the setup user administrator account must be a member of the **db_owner** role. 
+- If you use any PowerShell operations that affect a database, the setup user administrator account must be a member of the **db_owner** role or **sysadmin** fixed server role.
     
-- This account must be assigned to the **securityadmin** and **dbcreator** SQL Server security roles during setup and configuration. 
+- This account must be assigned to the **securityadmin** and **dbcreator** fixed server roles during setup and configuration or the **sysadmin** fix server role in SQL Server.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/account-permissions-and-security-settings-in-sharepoint-2013.md]:
```suggestion
- This account must be assigned to the **securityadmin** and **dbcreator** fixed server roles during setup and configuration or the **sysadmin** fixed server role in SQL Server.
```

**Comment by scanum** [comment]
@Techwriter40  Changes look correct. Could you please review and merge? Thanks!

**Pull request merged**